backend/test/fixtures/pbmc3k-genesets.csv

```diff
@@ -1,16 +1,21 @@
 # Test fixture
 gene_set_name,gene_set_description,gene_symbol,gene_description
-first gene set name,,F5, a gene_description
+first gene set name,a description,F5, a gene_description
 first gene set name,a description,NO_SUCH_GENE, non-existent gene
 first gene set name,a description,F5, duplicate gene
-first gene set name, a description,SUMO3,
-first gene set name,,SRM,
-second gene set,,RER1
-second gene set,,SIK1
-third gene set,,NO_SUCH_GENE
+first gene set name,a description,SUMO3,
+first gene set name,a description,SRM,
+second_gene_set,,RER1,
```

Comment:
line 8 and 9 were valid (ie missing a comma) and were there to test that we correctly handle it.  What was the reason for this change?